src/libraries/System.Security.Cryptography.X509Certificates/src/System/Security/Cryptography/X509Certificates/X509Certificate2.cs

```diff
@@ -837,7 +872,7 @@ public static X509Certificate2 CreateFromEncryptedPemFile(string certPemFilePath
         /// <para>
         ///   The SubjectPublicKeyInfo from the certificate determines what PEM labels are accepted for the private key.
         ///   For RSA certificates, accepted private key PEM labels are "RSA PRIVATE KEY" and "PRIVATE KEY".
-        ///   For ECDSA certificates, accepted private key PEM labels are "EC PRIVATE KEY" and "PRIVATE KEY".
+        ///   For ECDSA and ECDH certificates, accepted private key PEM labels are "EC PRIVATE KEY" and "PRIVATE KEY".
```

Comment:
Will XML doc edits eventually propogate back up to docs, or do I need to open a new PR after this one to modify existing docs?

Reply:
> Will XML doc edits eventually propogate back up to docs, or do I need to open a new PR after this one to modify existing docs?

Unfortunately, they don't propagate.  There's an import tool that imports the doc comments to the docs site for members which are entirely lacking documentation; but further changes happen in that repository and don't flow back to here... so updates here can't sensibly flow back to the docs site.

So, yeah, it'll need a manual PR to the docs site.  (I don't think it would need any sort of "starting in .NET 6.0" for this particular change)